helm/cluster-cloud-director/values.schema.json

```diff
@@ -441,6 +500,26 @@
                 "highest"
             ]
         },
+        "registry": {
+            "type": "object",
+            "properties": {
+                "configure": {
+                    "type": "boolean"
+                },
+                "defaultMirrors": {
+                    "$ref": "#/$defs/registryMirrors"
+                },
+                "extraMirrors": {
+                    "$ref": "#/$defs/registryMirrors"
+                },
+                "defaultConfigs": {
+                    "$ref": "#/$defs/registryConfigs"
+                },
+                "extraConfigs": {
+                    "$ref": "#/$defs/registryConfigs"
+                }
```

Comment:
title and description would be appreciated here. Especially since the distinction between default and extra would then become much more clear.